.travis.yml

```diff
@@ -1,13 +1,26 @@
 language: dart
 
-dart:
-  - 1.24.3
-  - stable
-
 # Re-use downloaded pub packages everywhere.
 cache:
   directories:
     - $HOME/.pub-cache
 
-script:
-  - ./validate.sh
+jobs:
+  include:
+    - dart: 2.2.0
```

Comment:
Why are we using `2.2.0` and not `2.4.1`?

Reply:
Since this is OSS, I wanted to keep the minimum as low as possible